pkgs/tools/package-management/harmonia/default.nix

```diff
@@ -9,16 +9,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "harmonia";
-  version = "0.2.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
-    owner = "helsinki-systems";
+    owner = "nix-community";
```

Comment:
Describe this change on the commit body message

Reply:
Linked the issue.